app/Actions/Diagnostics/Configuration.php

```diff
@@ -21,25 +18,22 @@ public function __construct(ConfigFunctions $configFunctions)
 	 * Return the config pieces of information of the Lychee installation.
 	 * Note that some information such as password and username are hidden.
 	 *
-	 * @return array
+	 * @return string[] array of messages
+	 *
+	 * @throws QueryBuilderException
 	 */
 	public function get(): array
 	{
-		// Declare
 		$configs = [];
 
-		try {
-			// Load settings
-			$settings = $this->configFunctions->min_info();
-			foreach ($settings as $key => $value) {
-				if (!is_array($value) && !is_null($value)) {
-					$configs[] = $this->line($key . ':', $value);
-				} elseif (is_null($value)) {
-					$configs[] = 'Error: ' . $key . ' has a NULL value!';
-				}
+		// Load settings
+		$settings = $this->configFunctions->min_info();
+		foreach ($settings as $key => $value) {
+			if (!is_array($value) && !is_null($value)) {
+				$configs[] = parent::line($key . ':', $value);
```

Comment:
```suggestion
				$configs[] = Diagnostics::line($key . ':', $value);
```
Just for uniformity with the other diagnostics files.

Reply:
I am not sure, if I have written that code or if I only moved it around. Anyway, we can change it nonetheless.